src/Mvc/Mvc.RazorPages/src/PageLoggerExtensions.cs

```diff
@@ -27,6 +32,26 @@ static PageLoggerExtensions()
         {
             // These numbers start at 101 intentionally to avoid conflict with the IDs used by ResourceInvoker.
 
+            _modelFactoryExecuting = LoggerMessage.Define<string, string>(
+                LogLevel.Debug,
+                new EventId(101, "ExecutingModelFactory"),
+               "Executing model factory for page {Page} ({AssemblyName})");
```

Comment:
I'd like the see the text of this message (and the method/field names) updated to use "page model factory" instead of "model factory" - that's the correct term, and it would be possible to confuse to the model term with some other concepts.